warehouse/query-core/src/main/java/datawave/query/jexl/visitors/RangeConjunctionRebuildingVisitor.java

```diff
@@ -138,8 +138,8 @@ protected JexlNode expandIndexBoundedRange(LiteralRange range, ASTReference curr
             return RebuildingVisitor.copy(currentNode);
         }
         
-        JexlNode orNode = JexlNodeFactory.createNodeTreeFromFieldsToValues(JexlNodeFactory.ContainerType.OR_NODE, new ASTEQNode(ParserTreeConstants.JJTEQNODE),
-                        currentNode, fieldsToTerms, expandFields, expandValues, false);
+        JexlNode orNode = JexlNodeFactory.createNodeTreeFromFieldsToValues(JexlNodeFactory.ContainerType.OR_NODE, false, currentNode, fieldsToTerms,
```

Comment:
Is there a need to track the negated state or is false always acceptable here?

Reply:
If we're expanding a negated node it determines how the resulting expansion is generated, so yes